packages/front-end/components/Experiment/EditDOMMutationsModal.tsx

```diff
@@ -0,0 +1,133 @@
+import { VisualChange } from "back-end/types/visual-changeset";
+import { FC, useCallback, useState } from "react";
+import Code from "@/components/SyntaxHighlighting/Code";
+import Modal from "../Modal";
+
+const EditDOMMutatonsModal: FC<{
+  visualChange: VisualChange;
+  close: () => void;
+  onSave: (newVisualChange: VisualChange) => void;
+}> = ({ close, visualChange, onSave }) => {
+  const [newVisualChange, setNewVisualChange] = useState<VisualChange>(
+    visualChange
+  );
+
+  const deleteCustomJS = useCallback(() => {
```

Comment:
nit: useCallback for these seems unnecessary, as it's a very inexpensive operation

Reply:
That's fair. It comes from a force of habit. I don't equate `useCallback`  as a thing to use when something is expensive but rather when 'we don't need to redefine this function per render'